CoffeeChat/Controller/EditDemographicsViewController.swift

```diff
@@ -357,8 +361,9 @@ extension EditDemographicsViewController: UIImagePickerControllerDelegate,
 
     func imagePickerController(_ picker: UIImagePickerController,
                                didFinishPickingMediaWithInfo info: [UIImagePickerController.InfoKey: Any]) {
-        let image = info[UIImagePickerController.InfoKey.originalImage] as! UIImage
-        profileImageView.image = image
+        guard let image = info[UIImagePickerController.InfoKey.originalImage] as? UIImage else { return }
+        let resizedImage = image.resize(toSize: CGSize(width: 40, height: 40))
```

Comment:
make this one line?

`profileImageView.image = image.resize(toSize: CGSize(width: 40, height: 40))`